From 61a5bd7c9ed92e3b12662ba985b2587bdb5a4840 Mon Sep 17 00:00:00 2001
From: Randy Barlow <randy@electronsweatshop.com>
Date: Sat, 14 Jan 2017 23:55:50 -0500
Subject: [PATCH] Fix a test to catch TypeError instead of Exception.

In PHP 7, TypeError is no longer an Exception but is an Error
instead. This commit adjusts a test case to catch TypeError instead
of Exception.
---
 tests/Evenement/Tests/EventEmitterTest.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/Evenement/Tests/EventEmitterTest.php b/tests/Evenement/Tests/EventEmitterTest.php
index 7eeaa3d..a553f5a 100644
--- a/tests/Evenement/Tests/EventEmitterTest.php
+++ b/tests/Evenement/Tests/EventEmitterTest.php
@@ -43,7 +43,7 @@ class EventEmitterTest extends \PHPUnit_Framework_TestCase
         try {
             $this->emitter->on('foo', 'not a callable');
             $this->fail();
-        } catch (\Exception $e) {
+        } catch (\TypeError $e) {
         }
     }
 
-- 
2.11.0

